index.js

```diff
@@ -49,37 +49,40 @@ class AddressFinder extends React.Component {
     const { chosenAddress, results, error, searchTerm } = this.state;
     return (
       <div>
-        <div>Address finder:</div>
+        <label htmlFor="search">Address finder:</label>
```

Comment:
a `for` attribute (`htmlFor` in JSX`) links a label to the corresponding input box by the `id` attribute.

- VoiceOver now reads out the label when focussed on the input field
- If you click on the label it puts focus on the input field. This is good for users who may struggle to click accurately.